Gruntfile.js

```diff
@@ -181,7 +181,9 @@ module.exports = function (grunt) {
                 npmLayout = grunt.file.read('./test/e2e/layouts/npm.tpl', {encoding: 'utf8'}),
                 sharedFunctions = grunt.file.read('./test/e2e/partials/shared-functions.js', {encoding: 'utf8'}),
                 testTpl = Handlebars.compile(content),
-                tplVars = {};
+                tplVars = {
+                  GENERATES: 'This file is auto-generated. Do not edit.'
```

Comment:
Did you mean `GENERATED` ?

Reply:
No, because this name will appear in the templates, which AREN'T generated (they generat_e_).  So I called it "GENERATES" - when you see that, you are looking at a file that generates another file.